api/server/sdk/cloud_backup.go

```diff
@@ -74,6 +74,9 @@ func (s *CloudBackupServer) Create(
 		Labels:         req.GetLabels(),
 	})
 	if err != nil {
+		if err == volume.ErrInvalidName {
+			return nil, status.Errorf(codes.AlreadyExists, "Failed to create backup: %v", err)
```

Comment:
Instead of using the same message, it should be something like "Backup already exists or invalid name"...